uxarray/grid/weights.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from numba import njit
+from uxarray.constants import INT_FILL_VALUE
+
+
+@njit
+def _face_weights_from_edge_magnitudes(
```

Comment:
The edges here are not normal edges here. They are not great circle arcs, instead, they are just constant latitude lines. And they're constructed by ourselves. 
So probably the name `edge` is not appropriate